pom.xml

```diff
@@ -66,6 +65,18 @@
     <artifactId>containerjfr-core</artifactId>
     <version>${com.redhat.rhjmc.containerjfr.core.version}</version>
   </dependency>
+  <!-- TODO move this into -core -->
```

Comment:
This is from jmc -> jmc-core right? I think there are some comments in code below for container-jfr -> container-jfr-core.

Reply:
Well, both. `container-jfr` shouldn't have any JMC dependencies, they should all be abstracted away into `container-jfr-core`. But also, a lot of those JMC dependencies are still "internal" JMC artifacts, and should be moved into jmc-core so that they can be properly published and consumed by other projects.

Reply:
Ah right you are. Thanks!